campaign-monitor-for-woocommerce/views/admin/connect.php

```diff
@@ -292,7 +292,7 @@
 
 
     <?php if (!\core\App::is_connected()) : ?>
-        <p>Campaign Monitor lets you manage your subscriber lists and email campaigns.<a href="https://www.campaignmonitor.com/signup/?utm_campaign=signup&utm_source=shopifyintegration&utm_medium=referral"> Send something beautiful today</a></p>
+        <p>Campaign Monitor lets you manage your subscriber lists and email campaigns.<a href="https://www.campaignmonitor.com?utm_source=woocommerce-plugin&utm_medium=referralsignup/?utm_campaign=signup&utm_source=shopifyintegration&utm_medium=referral"> Send something beautiful today</a></p>

```

Comment:
Should the new url goes to /signup/ as well? It seems there are multiple parameters in there.